packages/hashi/src/H5P/loadBinary.js

```diff
@@ -20,7 +20,7 @@ export default function(path) {
         if (xhr.readyState === 4) {
           if (xhr.status === 200 || xhr.status === 0) {
             try {
-              resolve(new Uint8Array(xhr.response));
+              resolve(xhr.response);
```

Comment:
Is this because the changes you've added mean that `xhr.response` is in a different format?

Reply:
No - just because the loadBinary is only used for loading resources into the zip file, and I preferred to delegate the coercion to Uint8Array to the ZipFile class instead, and leave this utility to simply load the binary data.